### PR TITLE
test: remove redundant page.close() calls

### PR DESCRIPTION
**Pull request by mxschmitt**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[x] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Comment by pamelafox** [comment]
Thanks @mxschmitt ! This repo is actually generated from a template, so the change would need to be made here:
https://github.com/kjaymiller/cookiecutter-relecloud/blob/main/%7B%7Bcookiecutter.__src_folder_name%7D%7D/src/tests/test_playwright.py
(And will then be propagated to quite a few repos - we use these same tests across Flask and Django as well)
I can make that change there to save you the work of another PR, if you'd like.

/CC @kjaymiller - our first PR on one of the generated repos!

File: demo_code/tests/test_playwright.py [hunk 1]
```diff
@@ -101,12 +98,10 @@ def test_destination_options_have_cruises(page: Page, mock_functions_env, live_s
 
     for page_cruise in page_cruises:
         assert page_cruise.text_content() in cruises
```
Comment (mxschmitt) [inline demo_code/tests/test_playwright.py:100]:
```suggestion
    expect(page.locator(".list-group-item")).to_contain_text(cruises)
```

this should also work in theory, but needs to be tested before. then you can also remove the line 97.

**Comment by mxschmitt** [comment]
> Thanks @mxschmitt ! This repo is actually generated from a template, so the change would need to be made here: https://github.com/kjaymiller/cookiecutter-relecloud/blob/main/%7B%7Bcookiecutter.__src_folder_name%7D%7D/src/tests/test_playwright.py (And will then be propagated to quite a few repos - we use these same tests across Flask and Django as well) I can make that change there to save you the work of another PR, if you'd like.
> 
> /CC @kjaymiller - our first PR on one of the generated repos!

yeah would be amazing if you could take over, thanks! Just saw this repo got mentioned on Twitter.

File: demo_code/tests/test_playwright.py (hunk 1 above)
Comment (pamelafox) [reply demo_code/tests/test_playwright.py:100]: That results in errors like:

>       result = next(iter(done)).result()
E       playwright._impl._api_types.Error: Error: strict mode violation: locator(".list-group-item") resolved to 2 elements:
E           1) <a href="/cruise/3" class="list-group-item list-gro…>The Cold Planets Expedition</a> aka get_by_role("link", name="The Cold Planets Expedition")
E           2) <a href="/cruise/5" class="list-group-item list-gro…>The Grand Solar System Tour</a> aka get_by_role("link", name="The Grand Solar System Tour")
E

**Comment by pamelafox** [comment]
Change was made in the template generator repo and propagated to the templates, thanks!

**Pull request closed without merging**